docs/data/toolpad/core/components/dashboard-layout/dashboard-layout.md

```diff
@@ -159,20 +159,30 @@ Some possibly useful slots:
 
 - `appTitle`: allows you to customize the app title section in the layout header.
 
-- `toolbarActions`: allows you to add new items to the toolbar in the header, such as a search bar or button. The default `ThemeSwitcher` component can be imported and used if you wish to do so, as shown in the example below.
+- `toolbarAccount`: (deprecated, use toolbar) allows you to replace the default `Account` slot in the header.
```

Comment:
I would keep the documentation changes more minimal, not add a new demo and just update the "Slots" section to accurately reflect the changes that were made.
Also no need to restructure headings!